src/test/java/com/squareup/kotlinpoet/TypeSpecTest.kt

```diff
@@ -19,11 +19,15 @@ import com.google.common.collect.ImmutableMap
 import com.google.common.truth.Truth.assertThat
 import com.google.testing.compile.CompilationRule
 import com.squareup.kotlinpoet.KModifier.ABSTRACT
+import com.squareup.kotlinpoet.KModifier.DATA
+import com.squareup.kotlinpoet.KModifier.IN
+import com.squareup.kotlinpoet.KModifier.INNER
 import com.squareup.kotlinpoet.KModifier.INTERNAL
 import com.squareup.kotlinpoet.KModifier.PRIVATE
 import com.squareup.kotlinpoet.KModifier.PUBLIC
 import com.squareup.kotlinpoet.KModifier.VARARG
 import com.squareup.kotlinpoet.jvm.throws
+import org.eclipse.jdt.internal.compiler.parser.Parser.name
```

Comment:
Is this used?

Reply:
👀  I have no idea where that came from. Will remove

Reply:
d01abec